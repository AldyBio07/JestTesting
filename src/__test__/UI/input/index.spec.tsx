import Input from "@/components/UI/input";
import { render, screen } from "@testing-library/react";

describe("Input Component", () => {
    it("render input component", () => { 
        const { container } = render(<Input type="text" value=""/>);
        expect(container).toMatchSnapshot();
    });

    it("render textarea component", () => { 
        const { container } = render(<Input type="textarea" value=""/>);
        expect(container).toMatchSnapshot();
    });
});