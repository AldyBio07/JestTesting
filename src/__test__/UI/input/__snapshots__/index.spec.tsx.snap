// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component render input component 1`] = `
<div>
  <div>
    <input
      type="text"
      value=""
    />
  </div>
</div>
`;

exports[`Input Component render textarea component 1`] = `
<div>
  <div>
    <textarea
      type="textarea"
    />
  </div>
</div>
`;
